README.md

```diff
@@ -62,10 +62,6 @@ df.columns = ['x', 'y', 'z1']
 df['x2'] = df.x * df.x
 ```
 
-## Documentation
```

Comment:
Let me leave the readthedoc link somewhere else later if you guys are fine with it.

Reply:
i think we should still keep this until we have published docs. Otherwise it looks like we created a project without any docs and even without the realization we need docs.

Reply:
Okie. Let me keep it